sdk/botpress.d.ts

```diff
@@ -896,6 +897,8 @@ declare module 'botpress/sdk' {
     }
 
     cloud?: CloudConfig
+    isCloudBot?: boolean
```

Comment:
not a critique, just a question: 

Why is there an optional `cloud` config and another field `isCloudBot`? Is it possible that a bot has a cloud config, but is not a cloud bot ?

Reply:
Fair question! The [original goal](https://linear.app/botpress/issue/DEV-2026#comment-ac1d42ac) was two keys – `isCloudBot` and `cloudApiKey`.

Then after noticing that `cloud` was already present, I spoke with @michaelmass who suggested `cloud` config would replace `cloudApiKey`. He also suggested: 
> `isCloudBot` could be added at the root to be able to switch between a cloud bot and a regular bot without changing the cloud config.

Reply:
I fully agree with this answer. @michaelmass is a beast 🐅